bokeh/models/tiles.py

```diff
@@ -1,7 +1,6 @@
 from __future__ import absolute_import
 from ..plot_object import PlotObject
-
-from ..properties import (Any, Dict, Float, String, Int)
+from ..properties import (Any, Dict, Float, String, Int, Bool)
 
 class TileSource(PlotObject):
```

Comment:
At this moment Tile Source's job is to `manage tile service configuration info` and `construction of tile urls`.  Inheriting from PlotObject, it doesn't have direct relationship to `bokeh.models.sources.DataSource`